efficientdet/keras/efficientdet_keras.py

```diff
@@ -757,10 +832,19 @@ def call(self, inputs, training):
     feats = self.fpn_cells(feats, training)
 
     # call class/box output network.
-    class_outputs = self.class_net(feats, training)
-    box_outputs = self.box_net(feats, training)
-
-    return class_outputs, box_outputs
+    outputs = []
+    for head in config.heads:
+      if head == 'object_detection':
+        class_outputs = self.class_net(feats, training)
+        box_outputs = self.box_net(feats, training)
+        outputs.append(class_outputs)
+        outputs.append(box_outputs)
+
+      if head == 'segmentation':
+        seg_outputs = self.seg_net(feats, training)
+        outputs.append(seg_outputs)
+
+    return tuple(outputs)
```

Comment:
Now, the model could train object-detection and segmentation together.

Reply:
This is excellent!